package.json

```diff
@@ -21,27 +21,15 @@
     "fs": false
   },
   "scripts": {
-    "bootstrap": "ocular-bootstrap",
-    "postinstall": "echo postinstall",
+    "bootstrap": "yarn && ocular-bootstrap && npm run build",
     "start": "open https://luma.gl/docs/getting-started",
-    "clean": "ocular-clean",
     "build": "ocular-clean && (cd modules/constants && yarn pre-build) && ocular-build",
-    "cover": "ocular-test cover",
     "lint": "ocular-lint",
     "publish-beta": "ocular-publish version-only-beta",
     "publish-prod": "ocular-publish version-only-prod",
-    "version": "node scripts/verify-changelog.js && git add CHANGELOG.md",
     "test": "ocular-test",
-    "test-ci": "ocular-lint && ocular-test node && ocular-test cover",
-    "test-node": "ocular-test node",
-    "test-cover": "ocular-test cover",
-    "test-fast": "ocular-test fast",
-    "test-browser": "ocular-test browser",
-    "test-browser-headless": "ocular-test browser-headless | tap-spec",
+    "test-fast": "ocular-lint && ocular-test node",
```

Comment:
Why keep this one? Not really useful at this stage, and with latest Puppeteer / Chromium browser tests are blazing fast.